### PR TITLE
feat: update header for sidebar

### PR DESCRIPTION
**Pull request by alangsto**
## [COSMO-571](https://2u-internal.atlassian.net/browse/COSMO-571)

This pull request includes changes to the `Sidebar` component to update its styling and improve its functionality. The most important changes include adding a new header with a logo, modifying the separator styling, and updating the corresponding tests.

Old sidebar header:
Screenshot 2024-12-05 at 11 43 38 AM

New sidebar h
Screenshot 2024-12-05 at 11 43 45 AM
eader:

### CODE REVIEW

**Review by schenedx — APPROVED** [review]
👍

File: src/components/Sidebar/index.jsx [hunk 1]
```diff
@@ -116,7 +111,7 @@ const Sidebar = ({
           alt="close"
           aria-label="close"
           variant="primary"
-          invertColors={!disclosureAcknowledged}
+          invertColors
```
Comment (varshamenon4) [inline src/components/Sidebar/index.jsx:114]: Nice!

File: src/widgets/Xpert.test.jsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ const courseId = 'course-v1:edX+DemoX+Demo_Course';
 const unitId = 'block-v1:edX+DemoX+Demo_Course+type@unit+block@unit1';
 
 const assertSidebarElementsNotInDOM = () => {
-  expect(screen.queryByTestId('heading', { name: 'Hi, I\'m Xpert!' })).not.toBeInTheDocument();
+  expect(screen.queryByTestId('sidebar-xpert-header')).not.toBeInTheDocument();
```
Comment (varshamenon4) [inline src/widgets/Xpert.test.jsx:39]: Nice! I like that you made the test id more descriptive.

**Review by varshamenon4 — APPROVED** [review]
lgtm!

**Pull request merged**